src/components/base/containers/Tooltip.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import Overlay from 'react-overlays/lib/Overlay';
+import Transition from 'react-overlays/lib/Transition';
+import styled, { injectGlobal } from 'styled-components';
+import { colors } from '../../theme';
+
+injectGlobal(`
```

Comment:
What if you do:

````javascript
const StyledTransition = styled(Transition)`
  .transition-in { ... }
  .transition-out { ... }
`;
````
At the very least it might prevent the globals.